README.md

```diff
@@ -1,2 +1,79 @@
-# Hello
-A simple project to familiarize with tools and best practices in software development
+<a name="readme-top"></a>
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 MICROVERSE HELLO PROJECT <a name="about-project"></a>
+
+**microverse hello project** is a simple project to familiarize with tools and best practices in software development 
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+This project is built with HTML and CSS
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **Demonstrate how to add html markup**
+- **Demonstrate adding styles to html markup**
+- **Demonstrate linking CSS and HTML files**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+1. Clone this repository
+   > run 'git clone git@github.com:marvin-nyalik/Hello_Microverse.git'
+2. Open it in your preffered text editor
+   >Preferable Visual Studio Code
+3. Serve the index.html file in a local web server
+   >Install the live server extension in VSCode
+
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+👤 **MARVIN OTIENO**
+
+- GitHub: [@githubhandle](https://github.com/marvin-nyalik/)
+- Twitter: [@twitterhandle](https://twitter.com/NyalikMarvin)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/marvin-otieno-05ba83263/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+- [ ] **Add Javascript to make it interactive**
+- [ ] **Add important external links**
+- [ ] **Add a styling framework**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](https://github.com/marvin-nyalik/Hello_Microverse/blob/main/LICENSE.md) licensed.
+
```

Comment:
- As Requested by your last reviewer, Kindly add a LICENSE file in your repository, in the branch you're working in, not the main branch.